telemetry/telemetry.go

```diff
@@ -256,6 +293,38 @@ func (reportMgr *NPMReportManager) SendReport() error {
 	return nil
 }
 
+// SendReport will send DNC telemetry report to HostNetAgent.
+func (reportMgr *DNCReportManager) SendReport() error {
+	var body bytes.Buffer
+
+	httpc := &http.Client{}
+	json.NewEncoder(&body).Encode(reportMgr.Report)
+
+	log.Printf("Going to send Telemetry report to hostnetagent %v", reportMgr.ReportManager.HostNetAgentURL)
```

Comment:
move this to common,.i see lot of things in common in other send report functions